### ISSUE TITLE
Zappa certify command "not recognised"

### ISSUE THREAD

**Issue by Antman261** [Miserlou/Zappa#274]
Following the instructions in [lets encrypt with zappa](https://github.com/Miserlou/Zappa/blob/master/docs/domain_with_free_ssl_dns.md) is giving me this error when I call certify. I've tried `pip install zappa --upgrade` and it hasn't helped.

```
$ zappa certify
The command 'certify' is not recognized. Please supply a command to execute. Can be one of: deploy, init, invoke, manage, rollback, schedule, status, tail, undeploy, unschedule, update
```

**Comment by jayfk** [comment]
That's not in the PyPi release from 2016-08-11, you need to install Zappa from git.

**Comment by Miserlou** [comment]
Hey Antman!

Sorry about that - we've been developing on the `-master` branch, not `-dev` like we should be - I've been using the Let's Encrypt successfully, I think it's ready to get published once #267 is confirmed and I've written one or two more LE tests.

I think you can probably consider `certify` to be "experimental" at least for another version or two as it gets battle tests, it's currently in a "works for me" state, but as people have more specific use cases it may need a bit more work.
